DOC: "Creating a Python environment" in "Creating a development environment" needs update

### Pandas version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://pandas.pydata.org/docs/dev/)


### Location of the documentation

https://pandas.pydata.org/docs/dev/development/contributing_environment.html#creating-a-python-environment

### Documentation problem

My machine is a MacBookPro with chip Apple M1 Pro, I'm using conda, and I created an environment without Docker. I checked my version of `xcode-select` and it's `2395` which seems to be the latest version. I followed the instructions in "Creating a Python environment". When I ran `python -m pip install -e . --no-build-isolation --no-use-pep517` I got the following output with the following errors: 
```
(pandas-dev) ➜  pandas git:(main) python -m pip install -e . --no-build-isolation --no-use-pep517
Looking in indexes: https://pypi.org/simple, https://maren.westermann:****@repos.mytaxi.com/artifactory/api/pypi/pypi/simple
Obtaining file:///Users/maren/open_source/pandas
  Preparing metadata (setup.py) ... done
Requirement already satisfied: python-dateutil>=2.8.1 in /Users/maren/mambaforge/envs/pandas-dev/lib/python3.8/site-packages (from pandas==0+untagged.29904.g6db95e7) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /Users/maren/mambaforge/envs/pandas-dev/lib/python3.8/site-packages (from pandas==0+untagged.29904.g6db95e7) (2022.1)
Requirement already satisfied: numpy>=1.20.3 in /Users/maren/mambaforge/envs/pandas-dev/lib/python3.8/site-packages (from pandas==0+untagged.29904.g6db95e7) (1.22.4)
Requirement already satisfied: six>=1.5 in /Users/maren/mambaforge/envs/pandas-dev/lib/python3.8/site-packages (from python-dateutil>=2.8.1->pandas==0+untagged.29904.g6db95e7) (1.16.0)
Installing collected packages: pandas
  Attempting uninstall: pandas
    Found existing installation: pandas 1.4.3
    Uninstalling pandas-1.4.3:
      Successfully uninstalled pandas-1.4.3
  Running setup.py develop for pandas
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
xarray 2022.6.0 requires pandas>=1.2, but you have pandas 0+untagged.29904.g6db95e7 which is incompatible.
statsmodels 0.13.2 requires pandas>=0.25, but you have pandas 0+untagged.29904.g6db95e7 which is incompatible.
seaborn 0.11.2 requires pandas>=0.23, but you have pandas 0+untagged.29904.g6db95e7 which is incompatible.
pyreadstat 1.1.9 requires pandas>=1.2.0, but you have pandas 0+untagged.29904.g6db95e7 which is incompatible.
pandas-gbq 0.17.6 requires pandas>=0.24.2, but you have pandas 0+untagged.29904.g6db95e7 which is incompatible.
pandas-datareader 0.10.0 requires pandas>=0.23, but you have pandas 0+untagged.29904.g6db95e7 which is incompatible.
mapclassify 2.4.3 requires pandas>=1.0, but you have pandas 0+untagged.29904.g6db95e7 which is incompatible.
geopandas 0.11.1 requires pandas>=1.0.0, but you have pandas 0+untagged.29904.g6db95e7 which is incompatible.
fastparquet 0.8.1 requires pandas>=1.1.0, but you have pandas 0+untagged.29904.g6db95e7 which is incompatible.
db-dtypes 1.0.3 requires pandas<2.0dev,>=0.24.2, but you have pandas 0+untagged.29904.g6db95e7 which is incompatible.
Successfully installed pandas-0+untagged.29904.g6db95e7
```

In the documentation for the development version name the example `0.22.0.dev0+29.g4ad6d4d74` is given which seems quite old. I think the documentation should be updated here 1. because the current naming system doesn't seem to follow the old convention of including the version number in the name of the installed pandas package and 2. given the error messages it's not clear if the installation process worked correctly. 

I ran `pytest pandas` to check if everything works correctly and the result was the following: 
`4445 failed, 179821 passed, 1715 skipped, 5498 xfailed, 11 xpassed, 221 warnings, 168 errors`
I'm not sure if the number of test failures is due to incorrect installation or if the failing tests just need to be fixed. 

This is the content of my pandas-dev conda environment:
<details>

```
# packages in environment at /Users/maren/mambaforge/envs/pandas-dev:
#
# Name                    Version                   Build  Channel
abseil-cpp                20210324.2           hbdafb3b_0    conda-forge
aiobotocore               1.4.1              pyhd8ed1ab_0    conda-forge
aiohttp                   3.8.1            py38h33210d7_1    conda-forge
aioitertools              0.10.0             pyhd8ed1ab_0    conda-forge
aiosignal                 1.2.0              pyhd8ed1ab_0    conda-forge
alabaster                 0.7.12                     py_0    conda-forge
anyio                     3.6.1              pyhd8ed1ab_1    conda-forge
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
appnope                   0.1.3              pyhd8ed1ab_0    conda-forge
argon2-cffi               21.3.0             pyhd8ed1ab_0    conda-forge
argon2-cffi-bindings      21.2.0           py38h33210d7_2    conda-forge
arrow-cpp                 8.0.1           py38hf3a2c03_0_cpu    conda-forge
asn1crypto                1.5.1              pyhd8ed1ab_0    conda-forge
asttokens                 2.0.5              pyhd8ed1ab_0    conda-forge
asv                       0.5.1            py38h396b780_0    conda-forge
async-timeout             4.0.2              pyhd8ed1ab_0    conda-forge
attrs                     22.1.0             pyh71513ae_1    conda-forge
aws-c-cal                 0.5.11               h4530763_0    conda-forge
aws-c-common              0.6.2                h3422bc3_0    conda-forge
aws-c-event-stream        0.2.7               h9972306_13    conda-forge
aws-c-io                  0.10.5               hea86ef8_0    conda-forge
aws-checksums             0.1.11               h487e1a8_7    conda-forge
aws-sam-translator        1.49.0             pyhd8ed1ab_0    conda-forge
aws-sdk-cpp               1.8.186              hb86bbe7_3    conda-forge
aws-xray-sdk              2.10.0             pyhd8ed1ab_0    conda-forge
babel                     2.10.3             pyhd8ed1ab_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
beautifulsoup4            4.11.1             pyha770c72_0    conda-forge
black                     22.3.0             pyhd8ed1ab_0    conda-forge
bleach                    5.0.1              pyhd8ed1ab_0    conda-forge
blinker                   1.4                        py_1    conda-forge
blosc                     1.21.1               hd414afc_3    conda-forge
bokeh                     2.4.3              pyhd8ed1ab_1    conda-forge
boost-cpp                 1.74.0               h1cb353e_8    conda-forge
boto3                     1.17.106           pyhd8ed1ab_0    conda-forge
botocore                  1.20.106           pyhd8ed1ab_0    conda-forge
bottleneck                1.3.5            py38hd91e067_0    conda-forge
branca                    0.5.0              pyhd8ed1ab_0    conda-forge
brotli                    1.0.9                h1c322ee_7    conda-forge
brotli-bin                1.0.9                h1c322ee_7    conda-forge
brotlipy                  0.7.0           py38h33210d7_1004    conda-forge
bzip2                     1.0.8                h3422bc3_4    conda-forge
c-ares                    1.18.1               h3422bc3_0    conda-forge
c-compiler                1.4.2                h50ab0ce_0    conda-forge
ca-certificates           2022.6.15            h4653dfc_0    conda-forge
cachetools                5.2.0              pyhd8ed1ab_0    conda-forge
cairo                     1.16.0            h3e596be_1011    conda-forge
cctools                   973.0.1             h43ef6ec_10    conda-forge
cctools_osx-arm64         973.0.1             h8e51184_10    conda-forge
certifi                   2022.6.15        py38h10201cd_0    conda-forge
cffi                      1.15.1           py38h0e10f82_0    conda-forge
cfgv                      3.3.1              pyhd8ed1ab_0    conda-forge
cfitsio                   4.1.0                hd4f5c17_0    conda-forge
cfn-lint                  0.61.4             pyhd8ed1ab_0    conda-forge
cftime                    1.6.1            py38hd91e067_0    conda-forge
charset-normalizer        2.1.0              pyhd8ed1ab_0    conda-forge
clang                     13.0.1               hce30654_0    conda-forge
clang-13                  13.0.1          default_h2cfa9b4_0    conda-forge
clang_osx-arm64           13.0.1               hc97ff27_2    conda-forge
clangxx                   13.0.1          default_hf584372_0    conda-forge
clangxx_osx-arm64         13.0.1               h15d2725_2    conda-forge
click                     8.1.3            py38h10201cd_0    conda-forge
click-plugins             1.1.1                      py_0    conda-forge
cligj                     0.7.2              pyhd8ed1ab_1    conda-forge
cloudpickle               2.1.0              pyhd8ed1ab_0    conda-forge
colorama                  0.4.5              pyhd8ed1ab_0    conda-forge
compiler-rt               13.0.1               h94c6b0c_0    conda-forge
compiler-rt_osx-arm64     13.0.1               h44bac85_0    conda-forge
coverage                  6.4.3            py38hbe6f924_0    conda-forge
cpplint                   1.6.0              pyhd8ed1ab_0    conda-forge
cramjam                   2.5.0            py38h76609ba_0    conda-forge
cryptography              37.0.4           py38h530bd25_0    conda-forge
curl                      7.83.1               h2fcd78c_0    conda-forge
cxx-compiler              1.4.2                hf86a087_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
cython                    0.29.32          py38h396b780_0    conda-forge
cytoolz                   0.12.0           py38he5c2ac2_0    conda-forge
dask                      2022.8.0           pyhd8ed1ab_0    conda-forge
dask-core                 2022.8.0           pyhd8ed1ab_0    conda-forge
dataclasses               0.8                pyhc8e2a94_3    conda-forge
db-dtypes                 1.0.3              pyhd8ed1ab_0    conda-forge
debugpy                   1.6.0            py38hea55cf5_0    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
distlib                   0.3.5              pyhd8ed1ab_0    conda-forge
distributed               2022.8.0           pyhd8ed1ab_0    conda-forge
docker-py                 5.0.3            py38h10201cd_2    conda-forge
docker-pycreds            0.4.0                      py_0    conda-forge
docutils                  0.17.1           py38h10201cd_2    conda-forge
doit                      0.36.0                   pypi_0    pypi
ecdsa                     0.18.0             pyhd8ed1ab_1    conda-forge
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
et_xmlfile                1.0.1                   py_1001    conda-forge
execnet                   1.9.0              pyhd8ed1ab_0    conda-forge
executing                 0.9.1              pyhd8ed1ab_0    conda-forge
expat                     2.4.8                h6b3803e_0    conda-forge
fastavro                  1.5.4            py38he5c2ac2_0    conda-forge
fastparquet               0.8.1            py38hd91e067_0    conda-forge
feedparser                6.0.10             pyhd8ed1ab_0    conda-forge
filelock                  3.8.0              pyhd8ed1ab_0    conda-forge
fiona                     1.8.21           py38h6cbfdb8_2    conda-forge
flake8                    4.0.1              pyhd8ed1ab_2    conda-forge
flake8-bugbear            21.3.2             pyhd8ed1ab_0    conda-forge
flake8-comprehensions     3.7.0              pyhd8ed1ab_0    conda-forge
flask                     2.2.2              pyhd8ed1ab_0    conda-forge
flask_cors                3.0.10             pyhd3deb0d_0    conda-forge
flit-core                 3.7.1              pyhd8ed1ab_0    conda-forge
folium                    0.12.1.post1       pyhd8ed1ab_1    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.14.0               hfb34624_0    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fonttools                 4.34.4           py38he5c2ac2_0    conda-forge
freetype                  2.10.4               hd633e50_2    conda-forge
freexl                    1.0.6                h27ca646_0    conda-forge
frozenlist                1.3.1            py38hbe6f924_0    conda-forge
fsspec                    2021.11.0          pyhd8ed1ab_0    conda-forge
gcsfs                     2021.11.0          pyhd8ed1ab_0    conda-forge
gdal                      3.5.1            py38hd5b0be9_1    conda-forge
geopandas                 0.11.1             pyhd8ed1ab_0    conda-forge
geopandas-base            0.11.1             pyha770c72_0    conda-forge
geos                      3.11.0               h9a09cb3_0    conda-forge
geotiff                   1.7.1                hc898e3f_3    conda-forge
gettext                   0.19.8.1          h049c9fb_1008    conda-forge
gflags                    2.2.2             hc88da5d_1004    conda-forge
giflib                    5.2.1                h27ca646_2    conda-forge
gitdb                     4.0.9              pyhd8ed1ab_0    conda-forge
gitpython                 3.1.27             pyhd8ed1ab_0    conda-forge
glog                      0.6.0                h6da1cb0_0    conda-forge
gmp                       6.2.1                h9f76cd9_0    conda-forge
gmpy2                     2.1.2            py38hc185ee4_0    conda-forge
google-api-core           2.8.2              pyhd8ed1ab_0    conda-forge
google-api-core-grpc      2.8.2                hd8ed1ab_0    conda-forge
google-auth               2.10.0             pyh6c4a22f_0    conda-forge
google-auth-oauthlib      0.5.2              pyhd8ed1ab_0    conda-forge
google-cloud-bigquery     3.2.0              pyhd8ed1ab_0    conda-forge
google-cloud-bigquery-core 3.2.0              pyhd8ed1ab_0    conda-forge
google-cloud-bigquery-storage 2.11.0             pyh6c4a22f_0    conda-forge
google-cloud-bigquery-storage-core 2.11.0             pyh6c4a22f_0    conda-forge
google-cloud-core         2.3.2              pyhd8ed1ab_0    conda-forge
google-cloud-storage      2.4.0              pyh6c4a22f_0    conda-forge
google-crc32c             1.1.2            py38ha01337b_3    conda-forge
google-resumable-media    2.3.3              pyhd8ed1ab_0    conda-forge
googleapis-common-protos  1.56.4           py38h10201cd_0    conda-forge
graphql-core              3.2.1              pyhd8ed1ab_0    conda-forge
greenlet                  1.1.2            py38hea55cf5_2    conda-forge
grpc-cpp                  1.45.2               h6e39ad6_4    conda-forge
grpcio                    1.46.3           py38hd51562d_0    conda-forge
grpcio-status             1.46.3             pyhd8ed1ab_0    conda-forge
hdf4                      4.2.15               h0ef36ac_3    conda-forge
hdf5                      1.12.1          nompi_hf9525e8_104    conda-forge
heapdict                  1.0.1                      py_0    conda-forge
html5lib                  1.1                pyh9f0ad1d_0    conda-forge
hypothesis                6.47.1             pyhd8ed1ab_0    conda-forge
icu                       70.1                 h6b3803e_0    conda-forge
identify                  2.5.3              pyhd8ed1ab_0    conda-forge
idna                      3.3                pyhd8ed1ab_0    conda-forge
imagesize                 1.4.1              pyhd8ed1ab_0    conda-forge
importlib-metadata        4.11.4           py38h10201cd_0    conda-forge
importlib_metadata        4.11.4               hd8ed1ab_0    conda-forge
importlib_resources       3.3.1              pyhd8ed1ab_1    conda-forge
iniconfig                 1.1.1              pyh9f0ad1d_0    conda-forge
ipykernel                 6.15.1             pyh736e0ef_0    conda-forge
ipython                   8.4.0            py38h10201cd_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                7.7.1              pyhd8ed1ab_0    conda-forge
isort                     5.10.1             pyhd8ed1ab_0    conda-forge
itsdangerous              2.1.2              pyhd8ed1ab_0    conda-forge
jedi                      0.18.1             pyhd8ed1ab_2    conda-forge
jinja2                    3.0.3                    pypi_0    pypi
jmespath                  0.10.0             pyh9f0ad1d_0    conda-forge
joblib                    1.1.0              pyhd8ed1ab_0    conda-forge
jpeg                      9e                   he4db4b2_2    conda-forge
jschema-to-python         1.2.3              pyhd8ed1ab_0    conda-forge
json-c                    0.16                 hc449e50_0    conda-forge
json5                     0.9.5              pyh9f0ad1d_0    conda-forge
jsondiff                  2.0.0              pyhd8ed1ab_0    conda-forge
jsonpatch                 1.32               pyhd8ed1ab_0    conda-forge
jsonpickle                2.2.0              pyhd8ed1ab_0    conda-forge
jsonpointer               2.0                        py_0    conda-forge
jsonschema                3.2.0              pyhd8ed1ab_3    conda-forge
junit-xml                 1.9                pyh9f0ad1d_0    conda-forge
jupyter_client            7.3.4              pyhd8ed1ab_0    conda-forge
jupyter_core              4.11.1           py38h10201cd_0    conda-forge
jupyter_server            1.18.1             pyhd8ed1ab_0    conda-forge
jupyterlab                3.4.5              pyhd8ed1ab_0    conda-forge
jupyterlab_pygments       0.2.2              pyhd8ed1ab_0    conda-forge
jupyterlab_server         2.15.0             pyhd8ed1ab_0    conda-forge
jupyterlab_widgets        1.1.1              pyhd8ed1ab_0    conda-forge
jupyterlite               0.1.0b10                 pypi_0    pypi
kealib                    1.4.14               hca62043_4    conda-forge
kiwisolver                1.4.4            py38haa2428a_0    conda-forge
krb5                      1.19.3               hf9b2bbe_0    conda-forge
lcms2                     2.12                 had6a04f_0    conda-forge
ld64                      609                 h954701c_10    conda-forge
ld64_osx-arm64            609                 h6ce2ded_10    conda-forge
lerc                      4.0.0                h9a09cb3_0    conda-forge
libblas                   3.9.0           16_osxarm64_openblas    conda-forge
libbrotlicommon           1.0.9                h1c322ee_7    conda-forge
libbrotlidec              1.0.9                h1c322ee_7    conda-forge
libbrotlienc              1.0.9                h1c322ee_7    conda-forge
libcblas                  3.9.0           16_osxarm64_openblas    conda-forge
libclang-cpp13            13.0.1          default_h2cfa9b4_0    conda-forge
libcrc32c                 1.1.2                hbdafb3b_0    conda-forge
libcst                    0.4.7            py38h1101ec3_0    conda-forge
libcurl                   7.83.1               h2fcd78c_0    conda-forge
libcxx                    14.0.6               h04bba0f_0    conda-forge
libdap4                   3.20.6               h8510809_2    conda-forge
libdeflate                1.13                 h57fd34a_0    conda-forge
libedit                   3.1.20191231         hc8eb9b7_2    conda-forge
libev                     4.33                 h642e427_1    conda-forge
libevent                  2.1.10               hbae9a57_4    conda-forge
libffi                    3.4.2                h3422bc3_5    conda-forge
libgdal                   3.5.1                h125b1ee_1    conda-forge
libgfortran               5.0.0.dev0      11_0_1_hf114ba7_23    conda-forge
libgfortran5              11.0.1.dev0         hf114ba7_23    conda-forge
libglib                   2.72.1               ha1047ec_0    conda-forge
libgoogle-cloud           1.40.2               h46da962_0    conda-forge
libiconv                  1.16                 h642e427_0    conda-forge
libkml                    1.3.0             h893ab4d_1014    conda-forge
liblapack                 3.9.0           16_osxarm64_openblas    conda-forge
libllvm11                 11.1.0               h93073aa_3    conda-forge
libllvm13                 13.0.1               hfd59cb2_2    conda-forge
libnetcdf                 4.8.1           nompi_h96a3436_102    conda-forge
libnghttp2                1.47.0               he723fca_0    conda-forge
libopenblas               0.3.21          openmp_hcb59c3b_0    conda-forge
libpng                    1.6.37               h2c9beb0_3    conda-forge
libpq                     14.4                 h8ab49ba_0    conda-forge
libprotobuf               3.20.1               h332123e_0    conda-forge
librttopo                 1.1.0               h275bb25_11    conda-forge
libsodium                 1.0.18               h27ca646_1    conda-forge
libspatialindex           1.9.3                hbdafb3b_4    conda-forge
libspatialite             5.0.1               h47b1232_18    conda-forge
libssh2                   1.10.0               hb80f160_2    conda-forge
libthrift                 0.16.0               h95ff8ae_1    conda-forge
libtiff                   4.4.0                hd6ca8d7_3    conda-forge
libutf8proc               2.7.0                h3422bc3_0    conda-forge
libwebp                   1.2.3                h1e26fe3_1    conda-forge
libwebp-base              1.2.3                he4db4b2_2    conda-forge
libxcb                    1.13              h9b22ae9_1004    conda-forge
libxml2                   2.9.14               h035c1df_3    conda-forge
libxslt                   1.1.35               ha8fb468_0    conda-forge
libzip                    1.9.2                h8caee39_0    conda-forge
libzlib                   1.2.12               ha287fd2_2    conda-forge
llvm-openmp               14.0.4               hd125106_0    conda-forge
llvm-tools                13.0.1               hfd59cb2_2    conda-forge
llvmlite                  0.38.1           py38h8a5a59d_0    conda-forge
locket                    1.0.0              pyhd8ed1ab_0    conda-forge
lxml                      4.9.1            py38he5c2ac2_0    conda-forge
lz4                       4.0.0            py38h6b08218_2    conda-forge
lz4-c                     1.9.3                hbdafb3b_1    conda-forge
mapclassify               2.4.3              pyhd8ed1ab_0    conda-forge
markdown                  3.4.1              pyhd8ed1ab_0    conda-forge
markupsafe                2.1.1            py38h33210d7_1    conda-forge
matplotlib                3.5.2            py38h150bfb4_1    conda-forge
matplotlib-base           3.5.2            py38hd7f6e49_1    conda-forge
matplotlib-inline         0.1.3              pyhd8ed1ab_0    conda-forge
mccabe                    0.6.1                      py_1    conda-forge
mistune                   0.8.4           py38hea4295b_1005    conda-forge
moto                      3.1.16             pyhd8ed1ab_0    conda-forge
mpc                       1.2.1                h309154c_0    conda-forge
mpfr                      4.1.0                h6d7a090_1    conda-forge
msgpack-python            1.0.4            py38haa2428a_0    conda-forge
multidict                 6.0.2            py38h33210d7_1    conda-forge
munch                     2.5.0                      py_0    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
mypy                      0.971            py38he5c2ac2_0    conda-forge
mypy_extensions           0.4.3            py38h10201cd_5    conda-forge
natsort                   8.1.0              pyhd8ed1ab_0    conda-forge
nbclassic                 0.4.3              pyhd8ed1ab_0    conda-forge
nbclient                  0.6.6              pyhd8ed1ab_0    conda-forge
nbconvert                 6.5.2              pyhd8ed1ab_0    conda-forge
nbconvert-core            6.5.2              pyhd8ed1ab_0    conda-forge
nbconvert-pandoc          6.5.2              pyhd8ed1ab_0    conda-forge
nbformat                  5.4.0              pyhd8ed1ab_0    conda-forge
nbsphinx                  0.8.9              pyhd8ed1ab_0    conda-forge
ncurses                   6.3                  h07bb92c_1    conda-forge
nest-asyncio              1.5.5              pyhd8ed1ab_0    conda-forge
networkx                  2.8.5              pyhd8ed1ab_0    conda-forge
ninja                     1.11.0               hf86a087_0    conda-forge
nodeenv                   1.7.0              pyhd8ed1ab_0    conda-forge
notebook                  6.4.12             pyha770c72_0    conda-forge
notebook-shim             0.1.0              pyhd8ed1ab_0    conda-forge
nspr                      4.32                 hbdafb3b_1    conda-forge
nss                       3.78                 h1483a63_0    conda-forge
numba                     0.55.2           py38h25e2f74_0    conda-forge
numexpr                   2.8.3            py38h6724fcd_0    conda-forge
numpy                     1.22.4           py38he1fcd3f_0    conda-forge
numpydoc                  1.4.0              pyhd8ed1ab_1    conda-forge
oauthlib                  3.2.0              pyhd8ed1ab_0    conda-forge
odfpy                     1.4.1                      py_0    conda-forge
openapi-schema-validator  0.2.3              pyhd8ed1ab_0    conda-forge
openapi-spec-validator    0.4.0              pyhd8ed1ab_1    conda-forge
openjpeg                  2.4.0                h062765e_1    conda-forge
openpyxl                  3.0.9              pyhd8ed1ab_0    conda-forge
openssl                   1.1.1q               ha287fd2_0    conda-forge
orc                       1.7.5                h96f55be_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pandas                    0+untagged.29914.g08fd9c0           dev_0    <develop>
pandas-datareader         0.10.0             pyh6c4a22f_0    conda-forge
pandas-dev-flaker         0.5.0              pyhd8ed1ab_0    conda-forge
pandas-gbq                0.17.6             pyh6c4a22f_0    conda-forge
pandoc                    2.19                 hce30654_0    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
parquet-cpp               1.5.1                         2    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
partd                     1.2.0              pyhd8ed1ab_0    conda-forge
pathspec                  0.9.0              pyhd8ed1ab_0    conda-forge
patsy                     0.5.2              pyhd8ed1ab_0    conda-forge
pbr                       5.9.0              pyhd8ed1ab_0    conda-forge
pcre                      8.45                 hbdafb3b_0    conda-forge
pexpect                   4.8.0              pyh9f0ad1d_2    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.2.0            py38h3852883_0    conda-forge
pip                       22.2.2             pyhd8ed1ab_0    conda-forge
pixman                    0.40.0               h27ca646_0    conda-forge
platformdirs              2.5.2              pyhd8ed1ab_1    conda-forge
pluggy                    1.0.0            py38h10201cd_3    conda-forge
poppler                   22.04.0              hadf1f10_1    conda-forge
poppler-data              0.4.11               hd8ed1ab_0    conda-forge
postgresql                14.4                 heaa1073_0    conda-forge
pre-commit                2.20.0           py38h10201cd_0    conda-forge
proj                      9.0.1                h4c79c2b_1    conda-forge
prometheus_client         0.14.1             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.30             pyha770c72_0    conda-forge
proto-plus                1.20.6             pyhd8ed1ab_0    conda-forge
protobuf                  3.20.1           py38h396b780_0    conda-forge
psutil                    5.9.1            py38he5c2ac2_0    conda-forge
psycopg2                  2.9.3            py38hbfad3ef_0    conda-forge
pthread-stubs             0.4               h27ca646_1001    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
py                        1.11.0             pyh6c4a22f_0    conda-forge
pyarrow                   8.0.1           py38h535b32c_0_cpu    conda-forge
pyasn1                    0.4.8                      py_0    conda-forge
pyasn1-modules            0.2.7                      py_0    conda-forge
pycodestyle               2.8.0              pyhd8ed1ab_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pydata-google-auth        1.4.0              pyhd8ed1ab_0    conda-forge
pydata-sphinx-theme       0.8.0              pyhd8ed1ab_0    conda-forge
pyflakes                  2.4.0              pyhd8ed1ab_0    conda-forge
pygments                  2.12.0             pyhd8ed1ab_0    conda-forge
pyjwt                     2.4.0              pyhd8ed1ab_0    conda-forge
pymysql                   1.0.2              pyhd8ed1ab_0    conda-forge
pyopenssl                 22.0.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.9              pyhd8ed1ab_0    conda-forge
pyproj                    3.3.1            py38hb16ab4f_1    conda-forge
pyreadstat                1.1.9            py38ha509820_0    conda-forge
pyrsistent                0.18.1           py38h33210d7_1    conda-forge
pysocks                   1.7.1            py38h10201cd_5    conda-forge
pytables                  3.7.0            py38h37503a7_0    conda-forge
pytest                    7.1.2            py38h10201cd_0    conda-forge
pytest-asyncio            0.19.0             pyhd8ed1ab_0    conda-forge
pytest-cov                3.0.0              pyhd8ed1ab_0    conda-forge
pytest-cython             0.2.0              pyhd8ed1ab_0    conda-forge
pytest-forked             1.4.0              pyhd8ed1ab_0    conda-forge
pytest-xdist              2.5.0              pyhd8ed1ab_0    conda-forge
python                    3.8.13          h7e095e3_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-fastjsonschema     2.16.1             pyhd8ed1ab_0    conda-forge
python-jose               3.3.0              pyh6c4a22f_1    conda-forge
python-snappy             0.6.0            py38hb0889cc_2    conda-forge
python_abi                3.8                      2_cp38    conda-forge
pytorch                   1.12.1          cpu_py38h7afd69f_0    conda-forge
pytz                      2022.1             pyhd8ed1ab_0    conda-forge
pyu2f                     0.1.5              pyhd8ed1ab_0    conda-forge
pyupgrade                 2.37.3             pyhd8ed1ab_0    conda-forge
pyxlsb                    1.0.9              pyhd8ed1ab_0    conda-forge
pyyaml                    6.0              py38h33210d7_4    conda-forge
pyzmq                     23.2.0           py38hc61cae3_0    conda-forge
re2                       2022.06.01           h9a09cb3_0    conda-forge
readline                  8.1.2                h46ed386_0    conda-forge
requests                  2.28.1             pyhd8ed1ab_0    conda-forge
requests-oauthlib         1.3.1              pyhd8ed1ab_0    conda-forge
responses                 0.21.0             pyhd8ed1ab_0    conda-forge
rsa                       4.9                pyhd8ed1ab_0    conda-forge
rtree                     1.0.0            py38hb34dff0_1    conda-forge
s3fs                      2021.11.0          pyhd8ed1ab_0    conda-forge
s3transfer                0.4.2              pyhd8ed1ab_0    conda-forge
sarif-om                  1.0.4              pyhd8ed1ab_0    conda-forge
scikit-learn              1.1.2            py38h6e93f71_0    conda-forge
scipy                     1.9.0            py38h4f188a7_0    conda-forge
seaborn                   0.11.2               hd8ed1ab_0    conda-forge
seaborn-base              0.11.2             pyhd8ed1ab_0    conda-forge
send2trash                1.8.0              pyhd8ed1ab_0    conda-forge
setuptools                63.4.2           py38h10201cd_0    conda-forge
sgmllib3k                 1.0.0              pyh9f0ad1d_0    conda-forge
shapely                   1.8.2            py38h7253250_3    conda-forge
sigtool                   0.1.3                h7747421_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
sleef                     3.5.1                h156473d_2    conda-forge
smmap                     3.0.5              pyh44b312d_0    conda-forge
snappy                    1.1.9                h39c3846_1    conda-forge
sniffio                   1.2.0            py38h10201cd_3    conda-forge
snowballstemmer           2.2.0              pyhd8ed1ab_0    conda-forge
sortedcontainers          2.4.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.3.2.post1        pyhd8ed1ab_0    conda-forge
sphinx                    4.5.0              pyh6c4a22f_0    conda-forge
sphinx-copybutton         0.5.0              pyhd8ed1ab_0    conda-forge
sphinx-panels             0.6.0              pyhd8ed1ab_0    conda-forge
sphinx-toggleprompt       0.2.0                    pypi_0    pypi
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    2.0.0              pyhd8ed1ab_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.5              pyhd8ed1ab_2    conda-forge
sqlalchemy                1.4.40           py38hbe6f924_0    conda-forge
sqlite                    3.39.2               h40dfcc0_0    conda-forge
sshpubkeys                3.3.1              pyhd8ed1ab_0    conda-forge
stack_data                0.3.0              pyhd8ed1ab_0    conda-forge
statsmodels               0.13.2           py38hd91e067_0    conda-forge
tabulate                  0.8.10             pyhd8ed1ab_0    conda-forge
tapi                      1100.0.11            he4954df_0    conda-forge
tblib                     1.7.0              pyhd8ed1ab_0    conda-forge
terminado                 0.15.0           py38h10201cd_0    conda-forge
threadpoolctl             3.1.0              pyh8a188c0_0    conda-forge
tiledb                    2.9.4                hdd33e7b_0    conda-forge
tinycss2                  1.1.1              pyhd8ed1ab_0    conda-forge
tk                        8.6.12               he1e0b03_0    conda-forge
tokenize-rt               4.2.1              pyhd8ed1ab_0    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
toolz                     0.12.0             pyhd8ed1ab_0    conda-forge
tornado                   6.1              py38h33210d7_3    conda-forge
tqdm                      4.64.0             pyhd8ed1ab_0    conda-forge
traitlets                 5.3.0              pyhd8ed1ab_0    conda-forge
typed-ast                 1.5.4            py38he5c2ac2_0    conda-forge
types-pymysql             1.0.19             pyhd8ed1ab_0    conda-forge
types-python-dateutil     2.8.19             pyhd8ed1ab_0    conda-forge
types-pytz                2022.1.2           pyhd8ed1ab_0    conda-forge
types-setuptools          63.4.0             pyhd8ed1ab_0    conda-forge
typing-extensions         4.3.0                hd8ed1ab_0    conda-forge
typing_extensions         4.3.0              pyha770c72_0    conda-forge
typing_inspect            0.7.1              pyh6c4a22f_0    conda-forge
tzcode                    2022a                h1c322ee_0    conda-forge
tzdata                    2022a                h191b570_0    conda-forge
ukkonen                   1.0.1            py38h0f71543_2    conda-forge
unicodedata2              14.0.0           py38h33210d7_1    conda-forge
urllib3                   1.26.11            pyhd8ed1ab_0    conda-forge
virtualenv                20.16.3          py38h10201cd_0    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
websocket-client          1.3.3              pyhd8ed1ab_0    conda-forge
werkzeug                  2.2.2              pyhd8ed1ab_0    conda-forge
wheel                     0.37.1             pyhd8ed1ab_0    conda-forge
widgetsnbextension        3.6.1              pyha770c72_0    conda-forge
wrapt                     1.14.1           py38he5c2ac2_0    conda-forge
xarray                    2022.6.0           pyhd8ed1ab_1    conda-forge
xerces-c                  3.2.3                h1b0b642_5    conda-forge
xlrd                      2.0.1              pyhd8ed1ab_3    conda-forge
xlsxwriter                3.0.3              pyhd8ed1ab_0    conda-forge
xlwt                      1.3.0                      py_1    conda-forge
xmltodict                 0.13.0             pyhd8ed1ab_0    conda-forge
xorg-libxau               1.0.9                h27ca646_0    conda-forge
xorg-libxdmcp             1.1.3                h27ca646_0    conda-forge
xyzservices               2022.6.0           pyhd8ed1ab_0    conda-forge
xz                        5.2.5                h642e427_1    conda-forge
yaml                      0.2.5                h3422bc3_2    conda-forge
yarl                      1.7.2            py38h33210d7_2    conda-forge
zeromq                    4.3.4                hbdafb3b_1    conda-forge
zict                      2.2.0              pyhd8ed1ab_0    conda-forge
zipp                      3.8.1              pyhd8ed1ab_0    conda-forge
zlib                      1.2.12               ha287fd2_2    conda-forge
zstandard                 0.18.0           py38he5c2ac2_0    conda-forge
zstd                      1.5.2                hd705a24_3    conda-forge
```

</details>

### Suggested fix for documentation

1. Give a newer example of the pandas development version naming convention
2. Include that errors during the installation process are expected due to currently unresolved dependency conflict (unless this is due to an incorrect installation process on my machine). 
3. Include in the documentation under "[Running the test suite](https://pandas.pydata.org/docs/dev/development/contributing_codebase.html#running-the-test-suite)"that there is currently a number of test failures when running `pytest pandas` and that these need to be fixed (unless this is due to an incorrect installation process on my machine).

Thanks a lot for all your input! 

> would it be possible that you cloned the pandas git repo with a limited "depth"?

No, I didn't clone the pandas repo with a limited depth. 

I synced my copy with `upstream/main` and I am now on commit [60b4400](https://github.com/marenwestermann/pandas/commit/60b44004919d85446236a73edff83cad9fc7851e). The output of `git describe` is `v1.5.0.dev0-1285-g60b4400491`. I followed the instructions in "[Creating a Python environment](https://pandas.pydata.org/docs/development/contributing_environment.html#id4)" again and this time I got the following error: 
```
➜  pandas git:(main) conda env create -f environment.yml
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound:
  - pandoc
```

I investigated the issue and the problem comes from [conda-forge/pandoc-feedstock](https://github.com/conda-forge/pandoc-feedstock/commit/946f7654b173e8824a8489a1cec59e679bb444ed). The support for my processor (osx-arm64) has currently been disabled. It worked previously since I happened to install my local environment exactly during the period where the binary of the package was available on conda-forge.

I then moved `pandoc` to the pip install section and also added pip to the list of environment packages. I did this by modifying the `environment.yml` file as follows (from line 128):
```
  # build the interactive terminal
  - jupyterlab >=3.4,<4
  - pip
  - pip:
      - jupyterlite==0.1.0b10
      - sphinx-toggleprompt
      - pandoc
```

After creating a new environment using the modified environment file, I then ran:
```
make clean
python setup.py build_ext -j 10
python -m pip install -e . --no-build-isolation --no-use-pep517
``` 
and the installation was successful. My version of pandas is now this one: `1.5.0.dev0+1285.g60b4400491.dirty`. 
I then ran:
```
pytest pandas -n auto
``` 
and I got 44 test failures. Here are some examples of the failures I got: 

<details>

```
________________________________________________________________________ ERROR at setup of TestMySQLAlchemy.test_read_sql_parameter _________________________________________________________________________
[gw9] darwin -- Python 3.9.13 /Users/maren/mambaforge/envs/pandas-test/bin/python

self = <pymysql.connections.Connection object at 0x2c9cdc4c0>, sock = None

    def connect(self, sock=None):
        self._closed = False
        try:
            if sock is None:
                if self.unix_socket:
                    sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
                    sock.settimeout(self.connect_timeout)
                    sock.connect(self.unix_socket)
                    self.host_info = "Localhost via UNIX socket"
                    self._secure = True
                    if DEBUG:
                        print("connected using unix_socket")
                else:
                    kwargs = {}
                    if self.bind_address is not None:
                        kwargs["source_address"] = (self.bind_address, 0)
                    while True:
                        try:
>                           sock = socket.create_connection(
                                (self.host, self.port), self.connect_timeout, **kwargs
                            )

/Users/maren/mambaforge/envs/pandas-test/lib/python3.9/site-packages/pymysql/connections.py:613:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

address = ('localhost', 3306), timeout = 10, source_address = None

    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,
                          source_address=None):
        """Connect to *address* and return the socket object.

        Convenience function.  Connect to *address* (a 2-tuple ``(host,
        port)``) and return the socket object.  Passing the optional
        *timeout* parameter will set the timeout on the socket instance
        before attempting to connect.  If no *timeout* is supplied, the
        global default timeout setting returned by :func:`getdefaulttimeout`
        is used.  If *source_address* is set it must be a tuple of (host, port)
        for the socket to bind as a source address before making the connection.
        A host of '' or port 0 tells the OS to use the default.
        """

        host, port = address
        err = None
        for res in getaddrinfo(host, port, 0, SOCK_STREAM):
            af, socktype, proto, canonname, sa = res
            sock = None
            try:
                sock = socket(af, socktype, proto)
                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:
                    sock.settimeout(timeout)
                if source_address:
                    sock.bind(source_address)
                sock.connect(sa)
                # Break explicitly a reference cycle
                err = None
                return sock

            except error as _:
                err = _
                if sock is not None:
                    sock.close()

        if err is not None:
            try:
>               raise err

/Users/maren/mambaforge/envs/pandas-test/lib/python3.9/socket.py:844:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

address = ('localhost', 3306), timeout = 10, source_address = None

    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,
                          source_address=None):
        """Connect to *address* and return the socket object.

        Convenience function.  Connect to *address* (a 2-tuple ``(host,
        port)``) and return the socket object.  Passing the optional
        *timeout* parameter will set the timeout on the socket instance
        before attempting to connect.  If no *timeout* is supplied, the
        global default timeout setting returned by :func:`getdefaulttimeout`
        is used.  If *source_address* is set it must be a tuple of (host, port)
        for the socket to bind as a source address before making the connection.
        A host of '' or port 0 tells the OS to use the default.
        """

        host, port = address
        err = None
        for res in getaddrinfo(host, port, 0, SOCK_STREAM):
            af, socktype, proto, canonname, sa = res
            sock = None
            try:
                sock = socket(af, socktype, proto)
                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:
                    sock.settimeout(timeout)
                if source_address:
                    sock.bind(source_address)
>               sock.connect(sa)
E               ConnectionRefusedError: [Errno 61] Connection refused

/Users/maren/mambaforge/envs/pandas-test/lib/python3.9/socket.py:832: ConnectionRefusedError

During handling of the above exception, another exception occurred:

self = Engine(mysql+pymysql://root@localhost:3306/pandas), fn = <bound method Pool.connect of <sqlalchemy.pool.impl.QueuePool object at 0x2ca72ef40>>, connection = None

    def _wrap_pool_connect(self, fn, connection):
        dialect = self.dialect
        try:
>           return fn()

/Users/maren/mambaforge/envs/pandas-test/lib/python3.9/site-packages/sqlalchemy/engine/base.py:3361:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <sqlalchemy.pool.impl.QueuePool object at 0x2ca72ef40>

    def connect(self):
        """Return a DBAPI connection from the pool.

        The connection is instrumented such that when its
        ``close()`` method is called, the connection will be returned to
        the pool.

        """
>       return _ConnectionFairy._checkout(self)
```

</details>

I saw that the documentation has been further improved which is great to see. I ran `pytest pandas` again and all the test failures I got were related to SQL which was expected given that I didn't set up a database: 

<details>

```
=============================================================================== short test summary info ================================================================================
FAILED pandas/tests/io/test_sql.py::test_to_sql[None-mysql_pymysql_engine] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'l...
FAILED pandas/tests/io/test_sql.py::test_to_sql[None-mysql_pymysql_conn] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'loc...
FAILED pandas/tests/io/test_sql.py::test_to_sql[None-postgresql_psycopg2_engine] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (...
FAILED pandas/tests/io/test_sql.py::test_to_sql[None-postgresql_psycopg2_conn] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (12...
FAILED pandas/tests/io/test_sql.py::test_to_sql[multi-mysql_pymysql_engine] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'...
FAILED pandas/tests/io/test_sql.py::test_to_sql[multi-mysql_pymysql_conn] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'lo...
FAILED pandas/tests/io/test_sql.py::test_to_sql[multi-postgresql_psycopg2_engine] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" ...
FAILED pandas/tests/io/test_sql.py::test_to_sql[multi-postgresql_psycopg2_conn] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (1...
FAILED pandas/tests/io/test_sql.py::test_to_sql_exist[replace-1-mysql_pymysql_engine] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for use...
FAILED pandas/tests/io/test_sql.py::test_to_sql_exist[replace-1-mysql_pymysql_conn] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user ...
FAILED pandas/tests/io/test_sql.py::test_to_sql_exist[replace-1-postgresql_psycopg2_engine] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "l...
FAILED pandas/tests/io/test_sql.py::test_to_sql_exist[replace-1-postgresql_psycopg2_conn] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "loc...
FAILED pandas/tests/io/test_sql.py::test_to_sql_exist[append-2-mysql_pymysql_engine] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user...
FAILED pandas/tests/io/test_sql.py::test_to_sql_exist[append-2-mysql_pymysql_conn] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user '...
FAILED pandas/tests/io/test_sql.py::test_to_sql_exist[append-2-postgresql_psycopg2_engine] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "lo...
FAILED pandas/tests/io/test_sql.py::test_to_sql_exist[append-2-postgresql_psycopg2_conn] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "loca...
FAILED pandas/tests/io/test_sql.py::test_to_sql_exist_fail[mysql_pymysql_engine] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'ro...
FAILED pandas/tests/io/test_sql.py::test_to_sql_exist_fail[mysql_pymysql_conn] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root...
FAILED pandas/tests/io/test_sql.py::test_to_sql_exist_fail[postgresql_psycopg2_engine] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localh...
FAILED pandas/tests/io/test_sql.py::test_to_sql_exist_fail[postgresql_psycopg2_conn] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhos...
FAILED pandas/tests/io/test_sql.py::test_read_iris[mysql_pymysql_engine] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'loc...
FAILED pandas/tests/io/test_sql.py::test_read_iris[mysql_pymysql_conn] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'local...
FAILED pandas/tests/io/test_sql.py::test_read_iris[postgresql_psycopg2_engine] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (12...
FAILED pandas/tests/io/test_sql.py::test_read_iris[postgresql_psycopg2_conn] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127....
FAILED pandas/tests/io/test_sql.py::test_to_sql_callable[mysql_pymysql_engine] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root...
FAILED pandas/tests/io/test_sql.py::test_to_sql_callable[mysql_pymysql_conn] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@...
FAILED pandas/tests/io/test_sql.py::test_to_sql_callable[postgresql_psycopg2_engine] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhos...
FAILED pandas/tests/io/test_sql.py::test_to_sql_callable[postgresql_psycopg2_conn] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost"...
FAILED pandas/tests/io/test_sql.py::test_default_type_conversion[mysql_pymysql_engine] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for us...
FAILED pandas/tests/io/test_sql.py::test_default_type_conversion[mysql_pymysql_conn] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user...
FAILED pandas/tests/io/test_sql.py::test_read_procedure[mysql_pymysql_engine] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'...
FAILED pandas/tests/io/test_sql.py::test_read_procedure[mysql_pymysql_conn] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'...
FAILED pandas/tests/io/test_sql.py::test_copy_from_callable_insertion_method[2-postgresql_psycopg2_engine] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection ...
FAILED pandas/tests/io/test_sql.py::test_copy_from_callable_insertion_method[2-postgresql_psycopg2_conn] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to...
FAILED pandas/tests/io/test_sql.py::test_copy_from_callable_insertion_method[Success!-postgresql_psycopg2_engine] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) conn...
FAILED pandas/tests/io/test_sql.py::test_copy_from_callable_insertion_method[Success!-postgresql_psycopg2_conn] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connec...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_read_sql_parameter - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_read_sql_named_parameter - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user '...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_to_sql_empty - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'local...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_create_table - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'local...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_drop_table - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'localho...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_roundtrip - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'localhos...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_execute_sql - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'localh...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_read_table - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'localho...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_read_table_columns - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_read_table_absent_raises - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user '...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_bigint - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'localhost'")
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_default_date_load - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_datetime_with_timezone - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'ro...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_datetime_with_timezone_roundtrip - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied fo...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_out_of_bounds_datetime - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'ro...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_naive_datetimeindex_roundtrip - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for u...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_date_parsing - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'local...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_datetime - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'localhost'")
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_datetime_NaT - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'local...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_datetime_date - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'loca...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_datetime_time - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'loca...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_mixed_dtype_insert - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_nan_numeric - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'localh...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_nan_fullcolumn - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'loc...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_nan_string - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'localho...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_to_sql_save_index - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_transactions - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'local...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_get_schema_create_table - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'r...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_dtype - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'localhost'")
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_notna_dtype - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'localh...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_double_precision - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'l...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_connectable_issue_example - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user ...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_to_sql_with_negative_npinf[input0] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied ...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_to_sql_with_negative_npinf[input1] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied ...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_to_sql_with_negative_npinf[input2] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied ...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_temporary_table - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'lo...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_invalid_engine - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'loc...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_options_sqlalchemy - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_options_auto - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'local...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_options_get_engine - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_get_engine_auto_error_message - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for u...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_default_type_conversion - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'r...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_read_sql_parameter - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'ro...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_read_sql_named_parameter - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for us...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_to_sql_empty - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'l...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_create_table - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'l...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_drop_table - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'loc...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_roundtrip - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'loca...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_execute_sql - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'lo...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_read_table - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'loc...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_read_table_columns - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'ro...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_read_table_absent_raises - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for us...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_bigint - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'localho...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_default_date_load - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'roo...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_datetime_with_timezone - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_datetime_with_timezone_roundtrip - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denie...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_out_of_bounds_datetime - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_naive_datetimeindex_roundtrip - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied f...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_date_parsing - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'l...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_datetime - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'local...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_datetime_NaT - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'l...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_datetime_date - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_datetime_time - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_mixed_dtype_insert - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'ro...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_nan_numeric - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'lo...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_nan_fullcolumn - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_nan_string - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'loc...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_to_sql_save_index - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'roo...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_get_schema_create_table - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for use...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_dtype - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'localhos...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_notna_dtype - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'lo...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_double_precision - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_connectable_issue_example - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for u...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_to_sql_with_negative_npinf[input0] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access den...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_to_sql_with_negative_npinf[input1] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access den...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_to_sql_with_negative_npinf[input2] - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access den...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_temporary_table - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_invalid_engine - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_options_sqlalchemy - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'ro...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_options_auto - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'root'@'l...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_options_get_engine - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for user 'ro...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_get_engine_auto_error_message - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied f...
ERROR pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_default_type_conversion - sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1698, "Access denied for use...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_read_sql_parameter - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" ...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_read_sql_named_parameter - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "local...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_to_sql_empty - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_create_table - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_drop_table - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0.0...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_roundtrip - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0.0....
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_execute_sql - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0....
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_read_table - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0.0...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_read_table_columns - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" ...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_read_table_absent_raises - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "local...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_default_type_conversion - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localh...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_bigint - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0.0.1),...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_default_date_load - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_datetime_with_timezone - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localho...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_datetime_with_timezone_roundtrip - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server a...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_out_of_bounds_datetime - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localho...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_naive_datetimeindex_roundtrip - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_date_parsing - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_datetime - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0.0.1...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_datetime_NaT - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_datetime_date - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127....
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_datetime_time - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127....
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_mixed_dtype_insert - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" ...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_nan_numeric - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0....
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_nan_fullcolumn - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_nan_string - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0.0...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_to_sql_save_index - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_transactions - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_get_schema_create_table - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localh...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_dtype - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0.0.1), ...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_notna_dtype - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0....
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_double_precision - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (1...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_connectable_issue_example - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "loca...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_to_sql_with_negative_npinf[input0] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_to_sql_with_negative_npinf[input1] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_to_sql_with_negative_npinf[input2] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_temporary_table - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (12...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_invalid_engine - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_options_sqlalchemy - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" ...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_options_auto - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_options_get_engine - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" ...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_get_engine_auto_error_message - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemy::test_schema_support - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_read_sql_parameter - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localho...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_read_sql_named_parameter - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "l...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_to_sql_empty - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (1...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_create_table - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (1...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_drop_table - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_roundtrip - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127....
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_execute_sql - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (12...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_read_table - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_read_table_columns - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localho...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_read_table_absent_raises - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "l...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_default_type_conversion - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "lo...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_bigint - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0.0...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_default_date_load - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhos...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_datetime_with_timezone - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "loc...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_datetime_with_timezone_roundtrip - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to serv...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_out_of_bounds_datetime - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "loc...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_naive_datetimeindex_roundtrip - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server ...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_date_parsing - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (1...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_datetime - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_datetime_NaT - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (1...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_datetime_date - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_datetime_time - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_mixed_dtype_insert - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localho...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_nan_numeric - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (12...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_nan_fullcolumn - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" ...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_nan_string - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_to_sql_save_index - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhos...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_get_schema_create_table - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "lo...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_dtype - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (127.0.0....
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_notna_dtype - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (12...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_double_precision - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_connectable_issue_example - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_to_sql_with_negative_npinf[input0] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to se...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_to_sql_with_negative_npinf[input1] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to se...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_to_sql_with_negative_npinf[input2] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to se...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_temporary_table - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost"...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_invalid_engine - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" ...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_options_sqlalchemy - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localho...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_options_auto - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (1...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_options_get_engine - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localho...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_get_engine_auto_error_message - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server ...
ERROR pandas/tests/io/test_sql.py::TestPostgreSQLAlchemyConn::test_schema_support - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" ...
=================================== 36 failed, 189009 passed, 1737 skipped, 3591 xfailed, 10 xpassed, 417 warnings, 168 errors in 4511.15s (1:15:11) ===================================
```

</details>

Two examples of full test failures can be seen here:
https://gist.github.com/marenwestermann/b27f060f2c95697fdcf8cf83402712a8
https://gist.github.com/marenwestermann/a52886e068ce0b7894051d8bbd9ddf06

I then ran `pytest pandas/tests/io/test_sql.py -m "not db"` in order to check if I get any test failures this way but all tests were successful. The command "not db" is mentioned in the documentation [here](https://pandas.pydata.org/docs/dev/development/contributing_codebase.html#running-the-test-suite) but only in relation to speed. Should it be explicitly mentioned in the documentation that `pytest pandas` will result in test failures if people didn't set up a database? 

I also ran `pytest pandas -n auto` to check if parallelisation causes problems but I currently don't get any additional test failures. (I did get an additional test failure related to memory usage recently but that seems to be fixed now). 

Should I open an issue for mentioning the tag problem in the contributing docs as suggested by @jorisvandenbossche or is there already one? 

Last but not least, sorry for the slow communication. I changed jobs recently and are slowly a bit more settled in which means I have a bit more time again now and can respond quicker. :)

--- a/doc/source/development/contributing_codebase.rst
+++ b/doc/source/development/contributing_codebase.rst
@@ -777,6 +777,14 @@ install pandas) by typing::
 
     pytest pandas
 
+.. note::
+
+    If a handful of tests don't pass, it may not be an issue with your pandas installation.
+    Some tests (e.g. some SQLAlchemy ones) require additional setup, others might start
+    failing because a non-pinned library released a new version, and others might be flaky
+    if run in parallel. As long as you can import pandas from your locally built version,
+    your installation is probably fine and you can start contributing!
+
 Often it is worth running only a subset of tests first around your changes before running the
 entire suite.
 

